Assets/Scripts/Combat/RangedWeapon.cs

```diff
@@ -1,10 +1,11 @@
+using System;
 using System.Collections;
 using UnityEditorInternal.Profiling.Memory.Experimental;
```

Comment:
Build will fail trying to find UnityEditorInternal. Can't finish build as long as this 'using' is here